source/MRMesh/MRMeshNormals.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 
+#include "MRId.h"
 #include "MRMeshFwd.h"
 #include "MRVector.h"
+#include "MRVector3.h"
```

Comment:
Can we avoid extra includes here?

Reply:
This file won't compile without this include anyway.